doc/changelog.rst

```diff
@@ -1,6 +1,61 @@
 Changelog
 =========
 
+0.3.0
+-----
+
+Expected Release on Febuary 11, 2022
+
+Description
+
+ - Update Redis version to 1.3.2 (PR162_)
+ - Streamline ``run`` and ``connect`` commands for Redis and RedisCluster (PR203_)
+ - Update handling of command replies (PR138_) (PR163_)
+ - Add SmartRedis namespace to ``srassert.h`` (PR191_)
+ - Expose Redis commands to the Python client (PR139_) (PR160_)
+ - Extend inverse CRC16 prefixing to all hash slots (PR161_)
+ - Refactor the dataset acknowledgement key for easier communication with ensambles (PR195_)
```

Comment:
Maybe change this to "Improve backend dataset representation to enable performance optimizations"